README.md

```diff
@@ -40,10 +40,12 @@ To learn more about the cloud-native architecture of ODC go to the [ODC document
 
 _Minimum system requirement per `outsystemscc` instance: 2 GB RAM, 2x 1GHz+ CPU._
 
-To install, use either the binary or Docker option. There are several advantages of running `outsystemscc` as a Docker image as opposed to a binary, if your system supports it:
+To install, use either the binary or Docker option. Run the binary on Linux, or use the Docker image on any OS that supports Docker. Running `outsystemscc` as a Docker image offers several advantages if your system supports it:
 
 * You always run the latest release. You don't need to reinstall each new release.
-* `outsystemscc` can run on Windows or any system that supports Docker.
+* You can run `outsystemscc` on Windows or any system that supports Docker:
+    * Use the `outsystemscc` Linux binary within Windows Subsystem for Linux(WSL) on Windows.
```

Comment:
Why are we mentioning the binary option in this section as it is dedicated to advantages of using Docker and we already explain that it is possible to use the binary on line 56?

I suggest to just keep the "Use the outsystemscc Linux container with Docker which is fully installed within WSL2 on Windows." subpoint in this part and, on line 56, we can mention that it works for both WSL1 and WSL2 -> if our goal is to explain further what is possible with each WSL version.

Also regarding the "Use the `outsystemscc` Linux container with Docker which is fully installed within WSL2 on Windows." line, this gives me the impression that Docker is already installed in WSL2 and I don't know if that's the case. Are we sure about that?

Reply:
So you need to explicitly install WSL in Windows.